artifacts/dashboard/ttyd-pod.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ttyd-cbf14e3f-f53a-4e0c-99e2-b39ebc0d221d
+  namespace: karmada-system
+spec:
+  containers:
+  - image: docker.io/sayem4604/ttyd:latest
```

Comment:
Can you provide with the original Dockerfile for building `docker.io/sayem4604/ttyd:latest` image, it will help our customer to clarify what we add in the image, also we can make some customization to to ttyd image, make it more convenient for karmada community.